include/librealsense2/dds/dds-device-server.cpp

```diff
@@ -85,7 +76,7 @@ class dds_device_server::dds_stream_server
         raw_image.width() = _image_header.width;
         raw_image.raw_data().assign( frame, frame + size );
 
-        _data_writer->write( &raw_image );
+        DDS_API_CALL_NO_THROW( _data_writer->write( &raw_image ) );
```

Comment:
Why no throw? Expecting a failure? Don't care? It's OK, just leave a comment because it's not obvious what the intent was.

Reply:
I didn't care, but I changed it to `DDS_API_CALL`, LRS will catch it and log it if needed.